hist/histv7/inc/ROOT/RHistData.hxx

```diff
@@ -245,10 +288,32 @@ public:
    /// Get the sum of weights.
    Weight_t GetSumOfSquaredWeights() const { return fSumWeights2; }
 
-   /// Merge with other RHistStatTotalSumOfSquaredWeights data, assuming same bin configuration.
+   /// Add an other RHistStatTotalSumOfSquaredWeights data, assuming same bin configuration.
    void Add(const RHistStatTotalSumOfSquaredWeights& other) {
       fSumWeights2 += other.fSumWeights2;
    }
+
+   /// Multiply by a scalar.
+   template <typename T>
+   void MultiplyByScalar(const T scalar) {
+      fSumWeights2 *= scalar;
```

Comment:
Math question: What are the expected semantics of multiplying by a scalar?

If this operation is equivalent to multiplying the weight of every previous fill by `scalar`, as I would expect, then shouldn't this be `fSumWeights2 *= scalar * scalar`?